index.html

```diff
@@ -48,7 +50,7 @@
                <div class="row hero-row">
                    <div class="col-12 col-sm-12 col-md-12 col-lg-6">
                        <div class="hero-text">
-                           <h1>Welcome to SLIIT FOSS Community</h1>
+                           <h1 style="text-shadow: 2px 2px 4px  rgba(0, 0, 0, 0.2);">Welcome to SLIIT FOSS Community</h1>
```

Comment:
@Shehanka can you remove the inline CSS and move it over to main CSS file please?